### DISCUSSION TITLE
Moving project to a GitHub organization

### DISCUSSION THREAD

**Discussion by nandgator** [astral-sh/rye#46] [General]
I think `rye` has great potential (esp. with the monorepo management). I'd love to see it developed and maintained by the community. I think moving the project to a GitHub organization would help more people to contribute.

**Comment by mitsuhiko** [comment]
I think for this is good as a discussion here. I'm generally open to a lot of this. I will probably share later today my current state of mind on this project and what I can see reasonable next steps are. An org very much sounds like something that should be on the table.

**Comment by jamesbraza** [comment]
As of 8/7/2024, it looks like `rye` is now part of `astral-sh` org: https://github.com/astral-sh/rye

So I think this is resolved

**Reply by charliermarsh** [reply]
True!
